registration/models.py

```diff
@@ -422,7 +422,7 @@ def _activate(self, profile, get_profile):
         """
 
         if not profile.user.is_active and not profile.activated:
-            site = Site.objects.get_current()
+            site = get_current_site()
```

Comment:
you have to pass the `request` in https://docs.djangoproject.com/en/1.8/ref/contrib/sites/#django.contrib.sites.shortcuts.get_current_site :)